src/libraries/System.Net.Security/src/System/Net/Security/NegotiateStreamPal.PlatformNotSupported.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.Security.Principal;
+
+namespace System.Net.Security
+{
+    //
+    // The class maintains the state of the authentication process and the security context.
+    // It encapsulates security context and does the real work in authentication and
+    // user data encryption with NEGO SSPI package.
+    //
+    internal static partial class NegotiateStreamPal
+    {
+        internal static IIdentity GetIdentity(NTAuthentication context)
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        internal static string QueryContextAssociatedName(SafeDeleteContext? securityContext)
+        {
+            throw new PlatformNotSupportedException(SR.net_nego_server_not_supported);
+        }
+
+        internal static void ValidateImpersonationLevel(TokenImpersonationLevel impersonationLevel)
+        {
+            throw new PlatformNotSupportedException();
+        }
```

Comment:
Why are these not annotated with Unsupported attribute

Reply:
@marek-safar Is there a point in these annotations on internal interface? I would expect them on `NegotiateStream` itself.

Reply:
I think it's useful for the tooling to ensure we didn't miss to annotate any call into PNSE or that new code does not incidentally take a dependency on it

Reply:
Annotated the code which the analyzer warned about, thanks